bin/kafka-run-class.sh

```diff
@@ -243,6 +243,14 @@ fi
 (( WINDOWS_OS_FORMAT )) && LOG_DIR=$(cygpath --path --mixed "${LOG_DIR}")
 KAFKA_LOG4J_CMD_OPTS="-Dkafka.logs.dir=$LOG_DIR $KAFKA_LOG4J_OPTS"
 
+# slf4j provider settings
+if [ -z "$SLF4J_PROVIDER" ]; then
```

Comment:
nit: Should this be KAFKA_SLF4J_PROVIDER? Or is this an idiomatic variable that we are following by convention?